pkg/lwip/contrib/lwip.c

```diff
@@ -146,19 +146,35 @@ void lwip_bootstrap(void)
         }
     }
 #elif defined(MODULE_ESP_WIFI)
+#ifdef MODULE_LWIP_IPV4
+    if (netif_add(&netif[0], IP4_ADDR_ANY, IP4_ADDR_ANY, IP4_ADDR_ANY,
+                  &_esp_wifi_dev, lwip_netdev_init, tcpip_input) == NULL) {
+        DEBUG("Could not add esp_wifi device\n");
+        return;
+    }
+#else   /* IPv6 */
```

Comment:
Usually we use the condition as comment for #else and #endif
```suggestion
#else /* MODULE_LWIP_IPV4 */
```